src/default_panels/StyleLayoutPanel.js

```diff
@@ -144,6 +144,16 @@ const StyleLayoutPanel = (props, {localize: _}) => (
       </PlotlySection>
       <PlotlySection name={_('Hover')}>
         <HovermodeDropdown label={_('Mode')} attr="hovermode">
+          <Dropdown
```

Comment:
this should be in Style/Traces, sorry, I don't think there's a global version

Reply:
ah never mind, there IS. but it should also be in Style/Traces (I think it's there also, right?)

Reply:
Done